src/os/ui/nodetoggle.js

```diff
@@ -49,17 +49,40 @@ os.ui.NodeToggleCtrl = function($scope, $element) {
    */
   this.element_ = $element;
 
+  /**
+   * @type {string}
+   */
+  this.scope_['expandedIcon'] = '';
```

Comment:
We should initialize these to the defaults to ensure icons are set if `item` isn't a `SlickTreeNode`.

Reply:
Thanks for the review.